best-practices/development-tools/docker/README.md

```diff
@@ -39,6 +39,10 @@ Docker Compose File Documentation:
   - [Nodejs](./nodejs)
   - [Django](./django)
 
+## Examples from Internal Projects:
+  - [node-boilerplate](https://github.com/Shift3/boilerplate-client-angular#docker)
```

Comment:
Not a big issue, I incline to unify this link with the bottom one by using https://github.com/Shift3/boilerplate-client-angular/blob/main/Dockerfile 
Unless it's in purpose.

Reply:
hmm good point, ill make the change.

Reply:
fixed via: 8ebeb097922d8c4531b2561a1d4b257683a4ed90